exercises/beer-song/canonical-data.json

```diff
@@ -10,32 +10,37 @@
           "cases": [
             {
               "description": "first generic verse",
-              "property": "verse",
-              "number": 99,
+              "property": "recite",
+              "startBottles": 99,
+              "endBottles": 99,
               "expected": "99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n"
             },
             {
               "description": "last generic verse",
-              "property": "verse",
-              "number": 3,
+              "property": "recite",
+              "startBottles": 3,
+              "endBottles": 3,
               "expected": "3 bottles of beer on the wall, 3 bottles of beer.\nTake one down and pass it around, 2 bottles of beer on the wall.\n"
             },
             {
-              "description": "verse 2",
-              "property": "verse",
-              "number": 2,
+              "description": "verse with 2 bottles",
+              "property": "recite",
+              "startBottles": 2,
+              "endBottles": 2,
               "expected": "2 bottles of beer on the wall, 2 bottles of beer.\nTake one down and pass it around, 1 bottle of beer on the wall.\n"
             },
             {
-              "description": "verse 1",
-              "property": "verse",
-              "number": 1,
+              "description": "verse with 1 bottle",
+              "property": "recite",
+              "startBottles": 1,
+              "endBottles": 1,
               "expected": "1 bottle of beer on the wall, 1 bottle of beer.\nTake it down and pass it around, no more bottles of beer on the wall.\n"
             },
             {
-              "description": "verse 0",
-              "property": "verse",
-              "number": 0,
+              "description": "verse with 0 bottles",
+              "property": "recite",
+              "startBottles": 0,
+              "endBottles": 0,
```

Comment:
There are 99 bottles on the wall at the end.

Maybe startBottles and endBottles doesn't make sense for this problem.

startBottles and takeDowns? perhaps.

(Trying to work out what's best here, not advocating changes just yet.)

Reply:
`takeBottles`? `bottlesToTake`?

Reply:
Whatever it is we're diverging from the common specification, but I think that is OK as this is a separate exercise with its own nuances.

Reply:
For the first two verses are you suggesting:
```
"startBottles": 99,
"takeDowns": 2,
```

or
```
"startBottles": 99,
"takeDowns": 98,
```

The first makes more sense to me but that's moving away in consistency with the other exercises unfortunately. If we're going to head down that direction, I think `numVerses` would be more explicit.

Reply:
Woops... just read your other comments you're one step ahead of me :)

Reply:
Should the words in the lyrics dictate that naming of the properties?

When we get down to zero beers on the wall, the only reason there ends up being 99 bottles of beer on the wall is because the lyrics say to go buy more beer.

To trigger a trip to the store, `endBottles` has to be zero.

Reply:
I'm actually suggesting: 

```
"startBottles": 99,
"takeDowns": 1,
```